meta-resin-morty/recipes-core/dropbear/files/atomic-hostkey.patch

```diff
@@ -0,0 +1,43 @@
+dropbearkey.c: Make host key generation atomic
```

Comment:
Sure. Created https://github.com/resin-os/meta-resin/pull/531 and https://github.com/resin-os/meta-resin/pull/530 so when these 2 PRs get merged I will remove this rework commit from the morty migration PR